homeassistant/components/light/lightwave.py

```diff
@@ -0,0 +1,85 @@
+"""
+Implements LightwaveRF lights.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.lightwave/
+"""
+from homeassistant.const import CONF_NAME
+from homeassistant.components.light import (
+    Light, ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS)
+from homeassistant.components.lightwave import LIGHTWAVE_LINK
+
+DEPENDENCIES = ['lightwave']
+
+MAX_BRIGHTNESS = 255
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info=None):
+    """Find and return LightWave lights."""
+    lights = []
```

Comment:
Add a guard clause here that checks if `discovery_info` is None and return if so.

Reply:
Done.